plutus-core/plutus-ir/src/PlutusIR/Compiler/Types.hs

```diff
@@ -51,16 +52,17 @@ data CompilationOpts = CompilationOpts {
     , _coDoSimplifierUnwrapCancel :: Bool
     , _coDoSimplifierBeta         :: Bool
     , _coDoSimplifierInline       :: Bool
+    , _coInlineHints              :: InlineHints PLC.Name (Provenance a)
     , _coProfile                  :: Bool
-    } deriving (Eq, Show)
+    } deriving (Show)
 
 makeLenses ''CompilationOpts
 
-defaultCompilationOpts :: CompilationOpts
-defaultCompilationOpts = CompilationOpts True False False False 8 True True True False
+defaultCompilationOpts :: CompilationOpts a
+defaultCompilationOpts = CompilationOpts True False False False 12 True True True mempty False
```

Comment:
Maybe extract `_soMaxSimplifierIterations` from `defaultSimplifyOpts` here?

Reply:
Mmm, but they're different. This is for the PIR compiler, the other is for the UPLC compiler, so (in principle) they shouldn't import each other. They end up being the same, but I think it's easier for now to just have them be quasi-independent. You could *imagine* wanting to tune them separately...

Reply:
Oh, makes sense, thanks.